app/app.js

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
-import { Provider } from 'react-redux';
-import { PersistGate } from 'redux-persist/lib/integration/react';
+import { I18nextProvider } from 'react-i18next';
 import LanguageProvider from '@atoms/LanguageProvider';
 import RootScreen from '@scenes/RootScreen';
+import i18n from '@app/i18n';
 import createStore from '@app/rootReducer';
-import { translationMessages } from './i18n';
-import 'react-native-gesture-handler';
 
+import { Provider } from 'react-redux';
+import { PersistGate } from 'redux-persist/lib/integration/react';
```

Comment:
can we please fix the import order

Reply:
placed as a placeholder for the other components that still are dependent on redux will be removed altogether